packages/fx-core/resource/strings.json

```diff
@@ -24,7 +24,7 @@
     "RegenerateArmTemplateConfirmNotice": "We will update your Azure resource template based on your resource and capability selection. The existing template file will be moved to backup folder under Azure resource template folder for your reference. We recommend to initialize your project workspace with git init to track all changes.",
     "LocalDebugTenantConfirmNotice": "You already run local debug for the Teams app in another tenant '%s'. You can sign out %s and sign in with the matched M365 account to re-use the previous local debug resource. If you really want to switch tenant for local debug, you can delete the localSettings.json file and re-run local debug.",
     "MigrateSuccessNotice": "Your project is now initialized with Teams Toolkit! You can debug your application by pressing F5. If you wish to host your application in Azure, we recommend you to re-create your project directly using the latest Teams Toolkit.",
-    "MigrationToArmAndMultiEnvMessage": "We will update your project in order to continue using the latest Teams Toolkit. We recommend to initialize your workspace with git for better tracking file changes.",
+    "MigrationToArmAndMultiEnvMessage": "Teams Toolkit will migrate your project to support the latest features in Teams Toolkit. This process will change the contents of the '.fx' folder. Backup this folder or put your project in source control if you want to revert your project back below Teams Toolkit 2.7.0.",
```

Comment:
Just one more change - "Teams Toolkit will migrate your project to support the latest features. This process ..."